beam_sink/sinks/mysql.py

```diff
@@ -73,6 +90,9 @@ def finish_bundle(self) -> None:
 
 
 class _PutFn(beam.DoFn):
+    """PutFn.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5069736)